File: HW 2 MyFirstApp/app/src/main/java/com/example/myfirstapp/MainActivity.java
package com.example.myfirstapp;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import com.example.myfirstapp.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        ActivityMainBinding binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
    }
}

File: HW4MyKotlinApp/app/src/main/java/com/example/hw4mykotlinapp/MainActivity.kt
package com.example.hw4mykotlinapp

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Switch
import android.widget.TextView

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val switch = findViewById<Switch>(R.id.switch1)
        val textView = findViewById<TextView>(R.id.textview_message)

        switch.setOnCheckedChangeListener { _, isChecked ->
            if (isChecked) {
                textView.text = resources.getString(R.string.switch_checked_text)
            } else {
                textView.text =resources.getString(R.string.switch_unchecked_text)
            }
        }
    }
}